utils/node_local_test.py

```diff
@@ -1453,6 +1449,10 @@ def _create_cont():
             rc = _create_cont()
 
     assert rc.returncode == 0, rc
+    if label:
+        assert label == rc.json['response']['container_label']
+    else:
+        assert 'container_label' not in rc.json['response'].keys()
```

Comment:
do we have any tests that are exercising this?
additionally, could we also add some C tests to verify that a middleware using the C API will not get this label in the properties?

Reply:
NLT tests call "daos" to create container many times both with and without labels. The checking code runs whenever a container is created in NLT tests. 
ok. I will write a simple c code test the API as a middleware. Thank you!

Reply:
A test case in C was added in src/tests/suite/daos_container.c.